### PR TITLE
Salary task

### PR DESCRIPTION
**Pull request by Shindred**

### CODE REVIEW

File: src/Salary.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public interface Salary {
+
+    int DAYS = 30;
+    int MONTHES = 1;
+    int HOURS = 8;
+    int HOURLYARRANGEMENT = 50;
```
Comment (malvitvik) [inline src/Salary.java:6]: Should this be a const for all employees?

File: src/Salary.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    int FIXEDARRANGEMENT = 20000;
```
Comment (malvitvik) [inline src/Salary.java:7]: The same as HOURLYRATE

File: src/Main.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+public class Main {
+
+    public static void main(String[] args) {
+
+	    HourlyRateEmployee john = new HourlyRateEmployee("John");
+	    FixedMonthlyRateEmployee jack = new FixedMonthlyRateEmployee("Jack");
+	    System.out.println("John's hourly salary is " + john.salary());
+	    System.out.println("Jack's fixed montly salary is " + jack.salary());
```
Comment (malvitvik) [inline src/Main.java]: If you store an emloyee name in variable - why you don't use it in **sout**? Like
john.getName() + "'s hourly salary is " + john.salary()

**Review by oleksandr-kazimirov — APPROVED** [review]
👍

**Pull request closed without merging**